components/editor/Editor.vue

```diff
@@ -204,3 +360,6 @@ onMounted(() => {
   });
 });
 </script>
+<style>
```

Comment:
Maybe we can try this. I haven't tested it.
```suggestion
<style scoped>
```